rascal-vscode-extension/src/ux/LibraryNavigator.ts

```diff
@@ -133,7 +161,7 @@ class RascalFSDirEntry extends RascalLibNode {
 
 class RascalFSFileEntry extends RascalLibNode {
     constructor(readonly fileName: string, readonly parentDir: vscode.Uri, parent: RascalLibNode) {
-        super(parentDir.with({path: posix.join(parentDir.path, fileName)}), vscode.TreeItemCollapsibleState.None, parent);
+        super(parentDir.with({path: posix.join(makeAbsolute(parentDir.path), fileName)}), vscode.TreeItemCollapsibleState.None, parent);
         this.command = <vscode.Command>{command:"vscode.open", title: "Open Rascal file", arguments: [this.resourceUri]};
```

Comment:
other files in the hierarchy might still be interesting for opening.

Reply:
I think this is a workaround for a bug of the uri parser in vscode, `tmp:///` as an empty string as path field. while it needs to be `/`.